content/telegraf/v1.20/administration/commands.md

```diff
@@ -0,0 +1,91 @@
+| `--config-directory <directory>` | Directory containing additional `*.conf` files                                                                                    |
+| `--watch-config`                 | Restart Telegraf on local config changes. Use either fs notifications (`inotify`) or polling (`poll`). Disabled by default        |
+| `--plugin-directory <directory>` | Directory containing `*.so` files to search recursively for plugins. Found plugins are loaded, tagged, and identified.            |
+| `--debug`                        | Enable debug logging.                                                                                                             |
+| `--input-filter <filter>`        | Filter inputs to enable. Separator is `:`.                                                                                        |
+| `--input-list`                   | Print available input plugins.                                                                                                    |
+| `--output-filter`                | Filter outputs to enable. Separator is `:`.                                                                                       |
+| `--output-list`                  | Print available output plugins.                                                                                                   |
+| `--pidfile <file>`               | File to write PID to.                                                                                                             |
+| `--pprof-addr <address>`         | pprof address to listen on. Disabled by default.                                                                                  |
+| `--processor-filter <filter>`    | Filter processors to enable. Separator is `:`.                                                                                    |
+| `--quiet`                        | Run in quiet mode.                                                                                                                |
+| `--section-filter <filter>`      | Filter config sections to output (`agent`, `global_tags`, `outputs`, `processors`, `aggregators` and `inputs`). Separator is `:`. |
+| `--sample-config`                | Print full sample configuration.                                                                                                  |
+| `--once`                         | Gather metrics once, write them, and exit.                                                                                        |
+| `--test`                         | Gather metrics once and print them.                                                                                               |
+| `--test-wait`                    | Number of seconds to wait for service inputs to complete in test or once mode.                                                    |
```

Comment:
```suggestion
| `--test-wait`                    | Number of seconds to wait for service inputs to complete in `test` or `once` mode.                                                    |
```

Reply:
would it be accurate to say service input plugins here? or add note to clarify that we refer to "configuration sections" as "plugins"...and "input plugins" as "inputs," or other terms we commonly use interchangeably?